### DISCUSSION TITLE
Permission denied to github-actions bot

### DISCUSSION THREAD

**Discussion by eamanj** [alshedivat/al-folio#653] [Q&A] [answered]
I can manually deploy my page using ./bin/deploy but every time I make a change, the automatic deployment fails with an error:

```
remote: Permission to eamanj/personal-webpage.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/eamanj/personal-webpage/': The requested URL returned error: 403
```

Any input is appreciated.

**Comment by charleslipku** [comment]
I run into the same issue, did you figure out the solution?

**Reply by eamanj** [reply]
No. I don't think it has to do with this package though. It must be something related to how github is set up.

**Comment by JohanvandenHeuvel** [comment, marked as answer]
For me changing the following setting in settings/actions/general fixed it:

![image](https://user-images.githubusercontent.com/24225735/167563991-e8012288-e5af-4148-be2a-398ad2a6a1ad.png)
